### PR TITLE
address some input and output issues with llama3 test

### PR DESCRIPTION
**Pull request by zjgarvey**
Unfortunately, it took me a bit of time to debug what was going wrong. With these changes, the test should pass on CPU (need to test GPU). 

1. There should only be two inputs (not 7 like flux). That being said, the lack of position_ids seems to generate an artificial third input for the onnx model which looks like it gets used as a negative padding for the output (it will slice across the sequence length dim from the back, e.g. passing [2] for the third input would return a logits tensor of shape [1, 512-2, 128256]). This should be 0 to match the pytorch output. 
2. I've included the output past-key-value pairs in the model info `forward` method, since the onnx model still has these outputs after tracing.
3. I've removed the rocm compile flags, since I don't know what they should be yet.
4. Added some comments.

### CODE REVIEW

**Comment by jinchen62** [comment]
Ah I see, I missed `externalize_inputs_threshold`. Thanks!!!
